client/components/popover/menu.jsx

```diff
@@ -17,6 +17,7 @@ class PopoverMenu extends Component {
 	static propTypes = {
 		autoPosition: PropTypes.bool,
 		isVisible: PropTypes.bool.isRequired,
+		isFocusEnabled: PropTypes.bool,
```

Comment:
> All I could think of was maybe renaming this prop to something like `focusOnMount`

Yeah, `focusOnMount` is a great choice!

> Another thought: why do we need to use `<Popover>`'s `onShow` prop at all? Why not just call `_onShow()` from its own `componentDidMount()` method?

I want to keep the responsibility of show and close on `Popover` because `PopoverMenu` is a component base on `Popover and IMO it should not handle the lifecycle again. Actually, I just fix the problems that `onShow` prop of `Popover` doesn't work correctly.